src/next-app/pages/services/[id].tsx

```diff
@@ -69,6 +76,36 @@ const ServicesDetail: NextPage = ({ service, parts, shop }: any) => {
     },
   });
 
+  const partForm = useFormik({
+    initialValues: {
+      name: '',
+      condition: 'new',
+      type: 'oem',
+      price: '',
+    },
+    validationSchema: partSchema,
+    onSubmit: async (values) => {
+      const valuesToSend = {
+        name: values.name,
+        condition: values.condition === 'New' ? 'new' : 'used',
+        type: values.type === 'OEM' ? 'oem' : 'aftermarket',
```

Comment:
That's weird because the only reason I added this is because it was sending 'Aftermarket' and not 'aftermarket' when I was sending the POST request. Let me check it out.

Reply:
Yeah, it does send it with the uppercase.

![image](https://user-images.githubusercontent.com/5843894/221302582-5dca99a0-b50d-4bda-8b0f-e4c35e93a3ce.png)

Reply:
Interesting 🤔 . I guess you can just do `.toLowerCase()` for this.

Reply:
But for me, it sends it on lowercase.
<img width="440" alt="Screenshot 2023-02-24 at 5 17 06 PM" src="https://user-images.githubusercontent.com/77129877/221303934-8eb687a9-8e29-411b-8f4c-56da86981147.png">

Reply:
But, the code you currently have:
```javascript
        condition: values.condition === 'New' ? 'new' : 'used',
        type: values.type === 'OEM' ? 'oem' : 'aftermarket',
```
always sends **used** and **aftermarket** to the backend.

Not sure why this is different on your machine 🤔

Reply:
That is very weird. I will update it to set it lowercase, then, yeah.

Reply:
Alright, it should send lowercase everytime now. Very weird difference between our machines, though.